pkg/operation/seed/seed.go

```diff
@@ -1036,6 +935,10 @@ func runCreateSeedFlow(
 
 	var (
 		g = flow.NewGraph("Seed cluster creation")
+		_ = g.Add(flow.Task{
+			Name: "Deploy Istio",
```

Comment:
```suggestion
			Name: "Deploying Istio",
```

(symmetry with other task names)